ci/test/mkpipeline.py

```diff
@@ -185,7 +185,13 @@ def have_paths_changed(globs: Iterable[str]) -> bool:
     diff = subprocess.run(
         ["git", "diff", "--no-patch", "--quiet", "origin/main...", *globs]
```

Comment:
Not related to this PR but technically you have to pass `"--"` before the user supplied globs to ensure they are not parsed as options